documentation/general/design/dash-sonic-hld.md

```diff
@@ -97,7 +97,7 @@ The following section captures at a high-level on the VNET packet flow. Detailed
 
 ## 2.1 Outbound packet processing pipeline
 	
-  ![dash-outbound](https://github.com/prsunny/DASH/blob/main/Assets/pkt_flow_outbound.png)
+  ![dash-outbound](https://github.com/Azure/DASH/blob/main/documentation/general/design/images/dash-hld-outbound-packet-processing-pipeline.svg)
```

Comment:
Hi, I suggest you use relative links which will always point to within a branch or fork being worked on. Otherwise, if the image gets edited (e.g. in a branch or fork), this URL will point to the main branch instead of the one being worked on. So, you could just change to `images/dash-hld-outbound-packet-processing-pipeline.svg`

Reply:
@prsunny You have addressed this issue described by @chrispsommers. You may want to resolve it, so the PR can be merged.

Reply:
Links need to be corrected.

Reply:
Done